docs/CONFIG.md

```diff
@@ -109,3 +109,72 @@ downloaded for local processing) will be stored per Python's
 which allow for configuration via the `TMPDIR`, `TEMP` or `TMP` env
 variables, and generally default to
 [something reasonable per your OS](https://docs.python.org/3/library/tempfile.html#tempfile.gettempdir).
+
+## Cloud credentials (e.g., S3/GCS/Google Sheets)
```

Comment:
Since database credentials come out of db-facts, the major difference made in switching the default creds provider from CredsViaLastPass and CredsViaEnv is how non-default GCP credentials are passed--this text should address that and help provide guidance.